apps/ledger-live-desktop/src/renderer/reducers/settings.ts

```diff
@@ -121,7 +121,7 @@ export const getInitialLocale = () => {
 };
 
 const INITIAL_STATE: SettingsState = {
-  hasCompletedOnboarding: false,
+  hasCompletedOnboarding: true,
```

Comment:
Not sure this default setting can be merged, @ofreyssinet-ledger wdyt? cc @KVNLS